libs/anglify/src/modules/snackbar/snackbar.interface.ts

```diff
@@ -0,0 +1,6 @@
+export type Position = 'center' | 'leading';
+
+export interface SnackbarSettings {
+  position?: Position;
+  stacked?: boolean;
+}
```

Comment:
The `SnackbarBarSettings` interface is not used anywhere. Please use it or remove it

Reply:
Oversight of mine 😓